### DISCUSSION TITLE
Change font size of track labels and annotation labels

### DISCUSSION THREAD

**Discussion by ALawaetz** [GMOD/jbrowse-components#3890] [Q&A] [answered]
I am using JBrowse2, the desktop version and I would like to change the font size of track labels and annotation labels. I am able to change the font size of the text in the track selector by adding this to the configuration file
    "theme": {
      "typography": {
        "fontSize": 16
      },

Is there a way that also changes the font size of track labels and annotation labels (gene names)?

I have attached a picture to illustrate what I mean.
[font_size_Jbrowse2_desktop.pdf](https://github.com/GMOD/jbrowse-components/files/12498977/font_size_Jbrowse2_desktop.pdf)

Andreas

**Comment by garrettjstevens** [comment, marked as answer]
Since you're using the desktop version, you can open the settings for the track and change the font there:

![image](https://github.com/GMOD/jbrowse-components/assets/25592344/1aa9ae63-767e-4566-8dfd-0561e0452ef8)

If you want to change it in the config file instead, you can add it in the "displays" section for the track in the config file:

```json
{
  "type": "FeatureTrack",
  "trackId": "gff3tabix_genes",
  "name": "GFF3Tabix genes",
  "assemblyNames": [
    "volvox"
  ],
  "adapter": {
    "type": "Gff3TabixAdapter",
    "gffGzLocation": {
      "locationType": "UriLocation",
      "uri": "volvox.sort.gff3.gz"
    },
    "index": {
      "location": {
        "locationType": "UriLocation",
        "uri": "volvox.sort.gff3.gz.tbi"
      }
    }
  },
  "displays": [
    {
      "type": "LinearBasicDisplay",
      "displayId": "gff3tabix_genes-LinearBasicDisplay",
      "renderer": {
        "type": "SvgFeatureRenderer",
        "labels": {
          "fontSize": 16
        }
      }
    }
  ]
}
```

**Reply by ALawaetz** [reply]
That is awesome! Thank you very much!
Is it possible to also change the font size of the track label?
[font_size_track_label.pdf](https://github.com/GMOD/jbrowse-components/files/12501159/font_size_track_label.pdf)

**Reply by garrettjstevens** [reply]
That particular label seems to be set differently from everything else, so it isn't adjustable from the theme right now. I've opened an issue here talking about this and other font issues: https://github.com/GMOD/jbrowse-components/issues/3893

You could also try changing the size of the font in your browser. In Chrome, for example, it's under chrome://settings/appearance. That seems to work better in JBrowse for some cases.

**Reply by ALawaetz** [reply]
Thanks for responding so quickly. So that solution is applicable to the web version of JBrowse. Is there a solution for the desktop version? I have read through the issue you have opened, but I don't understand the syntax of fontSize: theme.typography.body2.fontSize
Does that go into the config file under this?
{
  "configuration": {
    "rpc": {
      "defaultDriver": "WebWorkerRpcDriver",
      "drivers": {
        "MainThreadRpcDriver": {},
        "WebWorkerRpcDriver": {}
      }
    },
    "theme": {
      "typography": {
        "fontSize": 20
      },

**Reply by garrettjstevens** [reply]
Sorry, that's a note on some internal code we could possibly change, it's not something a user can apply. I'm not sure if there's currently a way change the default font size in the desktop app, I'll have to look into it some more.

**Reply by ALawaetz** [reply]
Thanks!
